### PR TITLE
Link to Rock the Vote (RTV) URL in the Voter Registration Badge in User Profile

### PR DESCRIPTION
**Pull request by Vmack**
### What does this PR do?

Adds Link to Rock the Vote (RTV) URL in the Voter Registration Badge in user profile

### Any background context you want to provide?

Provides user with a direct link to earning the voter registration badge  

### What are the relevant tickets/cards?

Refs [Pivotal ID #168223633](https://www.pivotaltracker.com/story/show/168223633)

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [ ] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

File: resources/assets/components/pages/AccountPage/Profile.js [hunk 1]
```diff
@@ -63,6 +61,7 @@ Profile.propTypes = {
     mobile: PropTypes.string,
     birthdate: PropTypes.string,
     voterRegistrationStatus: PropTypes.string,
+    id: PropTypes.string.isRequired,
```
Comment (katiecrane) [inline resources/assets/components/pages/AccountPage/Profile.js]: I thought these changes were already made, were you still on your old branch?

File: resources/assets/components/pages/AccountPage/BadgesTab.js [hunk 2]
```diff
@@ -108,8 +108,19 @@ const badgeModalContent = {
...
+        <a
+          href="https://vote.dosomething.org/?r=campaignID:8017,campaignRunID:8022,source:web,source_details:badges"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          register to vote
+        </a>{' '}
+        at your current address.
+      </span>
+    ),
```
Comment (katiecrane) [inline resources/assets/components/pages/AccountPage/BadgesTab.js]: This part looks great! Tested it on the review app and everything!

File: resources/assets/components/pages/AccountPage/Profile.js (hunk 1 above)
Comment (Vmack) [reply resources/assets/components/pages/AccountPage/Profile.js]: @katiecrane correct, my apologies!

Comment (weerd) [reply resources/assets/components/pages/AccountPage/Profile.js]: @Vmack if this wasn't a change you made, you may need to rebase from master to have these accounted for in this branch:

```
// from your current branch

1. git checkout master
2. git pull
3. git checkout profile-updates
4. git rebase master
5. ...fix merge conflicts if any...
6. git push -u origin profile-updates --force
```

Comment (Vmack) [reply resources/assets/components/pages/AccountPage/Profile.js]: Hey @weerd  so I corrected it. Let me know if it's good to go!

Comment (weerd) [reply resources/assets/components/pages/AccountPage/Profile.js]: I don't think anything actually changed! The file is still showing as being changed 😬 It should have updated the PR and leave only the single BadgesTab.js file changes.

**Pull request merged**